src/skeletonWriter.ts

```diff
@@ -15,6 +17,8 @@ export interface Edge {
 export class SkeletonWriter {
 
   static writeSkeleton(vertices: Vertex[], edges: Edge[], orientations: number[][], outputFilePath: string) {
+    fs.mkdirSync(path.dirname(outputFilePath), { recursive: true });
```

Comment:
Perhaps revisit this function call because we have unique timestamp-named tract folders now